src/main/scala/NetworkWordCount.scala

```diff
@@ -0,0 +1,34 @@
+import org.apache.spark.sql.SparkSession
+
+object NetworkWordCount {
```

Comment:
does it relate to the task of In-Stream Tweets classifier?

Reply:
It's from my previous spark experiment. Delete this now.